### ISSUE TITLE
@types/mocha broke mocha-typescript again

### ISSUE THREAD

**Issue by pana-cc** [pana-cc/mocha-typescript#98]
This: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/6bddfd9921357e6dea040dee8724717a6d3a1e86#commitcomment-29353340
Broke mocha-typescript.
Waiting for PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26523
After that mocha-typescript should retarget the new interfaces for augmentation.

**Comment by pana-cc** [comment]
I kind of don't know what to do. We should probably keep a record of "recommended/tested with @types/mocha and typescript versions"

**Comment by silkentrance** [comment]
@pana-cc They must not break stuff 👍 
Otherwise we would have to make this a locked version dependency and a peer dependency by that, so that the user can instead use a different version, e.g.

in our package.json

```
{
  "peerDependencies": {
    "@types/mocha": "5.2.0"
  }
}
```

and in the user's package.json

```
{
  "devDependencies": {
    "@types/mocha": "^6.4.0.fancy-stuff-here"
  }
}
```

See https://nodejs.org/en/blog/npm/peer-dependencies/.

Maybe this can help solve the problem.

**Comment by ChuckJonas** [comment]
does anyone know what the latest version of mocha this will work with?  Tried with `"mocha": "^5.2.0",` and it fails with a bunch of type errors

Namespace 'Mocha' has no exported member 'ITestDefinition'.

**Comment by silkentrance** [comment]
@ChuckJonas have your tried

```
    "@types/mocha": "^5.2.1",
```

This works fine for me.

**Comment by silkentrance** [comment]
@ChuckJonas and have you also upgraded to latest version of mocha-typescript?
```
    "mocha-typescript": "^1.1.16",
```

**Comment by ChuckJonas** [comment]
@silkentrance ya, its working now... I just had to fix my package lock so it actually pulled `5.2.0` (always gets me).

Thanks for the help

**Comment by silkentrance** [comment]
@ChuckJonas package-lock.json is really annoying 😁

**Comment by pana-cc** [comment]
This one is still blocking us:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26970

**Comment by pana-cc** [comment]
This should be resolved with v1.1.17
